amd/Dockerfile

```diff
@@ -0,0 +1,93 @@
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE}
+
+ARG ROCM_VERSION=5.3
+ARG AMDGPU_VERSION=5.3
+# Base
+RUN yum -y install git java-1.8.0-openjdk python; yum clean all
+
+# Enable epel-release repositories
```

Comment:
Could you add as comment the link of the source code that you fetch these dockerfile instructions please?

Reply:
> Could you add as comment the link of the source code that you fetch these dockerfile instructions please?
[
](https://github.com/ROCm/ROCm-docker/blob/master/dev/Dockerfile-centos-7-complete)

Reply:
Thx for the link! Do you mind to comment the link inside on Dockerfile? like:
```suggestion
# Install ROCm AMD from:
# https://github.com/ROCm/ROCm-docker/blob/master/dev/Dockerfile-centos-7-complete

# Enable epel-release repositories